README.md

```diff
@@ -27,7 +27,7 @@ The workflows can be installed by the meta chart or individually. Visit workflow
 
 To install workflows by the meta-chart, run:
 ```
-$ helm install orchestrator-workflows orchestrator-workflows/workflows
+$ helm install orchestrator-workflows orchestrator-workflows/workflows --namespace=sonataflow-infra
```

Comment:
with the current version of the operator, we can install workflows in any namespace. there is no longer a limitation to a specific namespace.

Reply:
ah, good thing
But do we want that? As there is no namespace specified in the manifests, the resources will go in the default namespace (or the current project), by adding this we make it obvious to user how they can easily set the namespace

Reply:
it can be any namespace, depending on the admin needs, but if this isn't the devmode type of deployment, then for workflows to run with persistency, it will work without any issues from the sonataflow-infra. This might not be the case for other workflows. There is a WIP from Jordi to address the persistence use-case: https://github.com/apache/incubator-kie-kogito-serverless-operator/pull/372

so for your question, do we want this PR? It isn't harmful, with non-devmode even recommended for now. So let's have it, or specify for a devmode deployment, specifying any namespace (or none, to use existing one) is supported